### PR TITLE
Feature/395 expose docs in ui

### PR DESCRIPTION
**Pull request by kaspersorensen**
Another inching towards fixing #395 :) This fixes the HiddenProperty part as well as the need to expose the docs more prominently in the desktop UI.

### CODE REVIEW

**Comment by kaspersorensen** [comment]
And I stumbled across a bad use of the Qualifier annotation in various property metadata annotations. Removed that aswell, but it doesn't have any meaning except for symbolically.

**Comment by kaspersorensen** [comment]
Error in travis seems unrelated, probably a glitch in the travis infra

File: desktop/ui/src/main/java/org/datacleaner/documentation/ComponentDocumentationBuilder.java [hunk 1]
```diff
@@ -109,9 +110,13 @@ public void write(ComponentDescriptor<?> componentDescriptor, OutputStream outpu
                         configuredProperties);
                 final List<ConfiguredPropertyDocumentationWrapper> propertyList = new ArrayList<>();
                 for (ConfiguredPropertyDescriptor property : properties) {
-                    ConfiguredPropertyDocumentationWrapper wrapper = new ConfiguredPropertyDocumentationWrapper(
-                            property);
-                    propertyList.add(wrapper);
+                    final HiddenProperty hiddenProperty = property.getAnnotation(HiddenProperty.class);
+                    // we do not show hidden properties in docs
+                    if (hiddenProperty == null) {
+                        ConfiguredPropertyDocumentationWrapper wrapper = new ConfiguredPropertyDocumentationWrapper(
```
Comment (LosD) [inline desktop/ui/src/main/java/org/datacleaner/documentation/ComponentDocumentationBuilder.java]: Let's add final now that we're reformatting anyway :)

**Comment by LosD** [comment]
LGTM. You can add a final if you feel like it :)

**Pull request merged**